rafay/resource_eks_cluster.go

```diff
@@ -4386,15 +4499,15 @@ func flattenSingleIAMServiceAccount(in *EKSClusterIAMServiceAccount) map[string]
 		obj["attach_policy_arns"] = toArrayInterface(in.AttachPolicyARNs)
 	}
 	if in.AttachPolicy != nil && len(in.AttachPolicy) > 0 {
-		//log.Println("type:", reflect.TypeOf(in.AttachPolicy))
+		log.Println("type:", reflect.TypeOf(in.AttachPolicy))
 		var json2 = jsoniter.ConfigCompatibleWithStandardLibrary
 		jsonStr, err := json2.Marshal(in.AttachPolicy)
 		if err != nil {
 			log.Println("attach policy marshal err:", err)
 		}
-		//log.Println("jsonSTR:", jsonStr)
```

Comment:
@gopikrishna-rafay Do we need this log ?

Reply:
No @chaitanya-rafay, I added while. There are some others comments, I will clean up in final PR